Feature/pl 6066

e136581 opened the pull request:
Updated OrienteringOmRettigheterUfoereDto api-model with harInnvilgetBarnetillegg

routsi commented on pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/vedlegg/OrienteringOmRettigheterUfoere.kt:
```diff
@@ -5,9 +5,10 @@ import no.nav.pensjon.brev.api.model.Kroner
 data class OrienteringOmRettigheterUfoereDto(
+    val avdoed_sivilstand: Sivilstand,
     val bruker_borINorge: Boolean,
+    val harInnvilgetBarnetillegg: Boolean,
```
Er denne lagt til fordi den skal legges til i mapping i pesys også taes i bruk i vedlegget?

hconhegg replied:
Ja

e136581 commented:
Updated OrienteringOmRettigheterUfoereDto api-model with harInnvilgetBarnetillegg and removed ufoeretrygdPerMaaned_barnetilleggGjeldende: Kroner?

Updated VedleggOrienteringOmRettigheter.kt and OrienteringOmRettigheterUfoere.kt with 2 new frases "vedleggPlikterUT13_001" and "vedleggPlikterUT14_001"
and changed text in phrase vedleggInnsynSakUTPesys_001 (PL-6062)
